src/main/scala/castalia/Main.scala

```diff
@@ -5,7 +5,7 @@ import akka.http.scaladsl.Http
 import akka.http.scaladsl.Http.ServerBinding
 import akka.http.scaladsl.model._
 import castalia.management.{Manager, ManagerService}
-import castalia.model.CastaliaConfig
+import castalia.model.Model.CastaliaConfig
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/scala-academy/Castalia/file/1940175477/issues/source?bid=2998213&fileBranchId=3101401#l8)